appveyor.yml

```diff
@@ -1,34 +1,44 @@
-# AppVeyor file
 # http://www.appveyor.com/docs/appveyor-yml
 
-version: "{build}"
-
 clone_depth: 10
 
+version: "{build}"
+
+# What combinations to test
 environment:
   matrix:
-    - nodejs_version: "5"
-    - nodejs_version: "4"
     - nodejs_version: "0.10"
+      platform: x86
+    - nodejs_version: "0.12"
+      platform: x86
+    - nodejs_version: "4"
+      platform: x64
+    - nodejs_version: "4"
+      platform: x86
+    - nodejs_version: "5"
+      platform: x86
```

Comment:
Have you even checked before commenting??

We normalize CI configs via grunt-contrib-internal and not per plugin.